crates/pipeline_manager/src/api/examples.rs

```diff
@@ -138,7 +138,7 @@ pub(crate) fn pipeline_config() -> PipelineConfig {
         config: ConnectorConfig::from_yaml_str(
             r#"
 transport:
-    name: kafka
+    name: kafka_input
```

Comment:
It seems redundant from the user perspective to have to add the `_input` suffix since this already goes to the `inputs` section of the config. Maybe we can figure out a way around this.

Reply:
The tricky part is that the kafka input and kafka ouput connector have different fields, as such making it strongly typed requires to differentiate them. In the inputs section, we can actually then drop the "is_input" part of it, and just check that the connector is an input (or output) connector.

Reply:
The API would also get more complicated if we would use optional fields of for example topic (output) and topics (input), with things like "If field A is specified, don't specify B".

Reply:
Maybe untagged could be a way to get around it, though would have to think about it more if that is desirable in terms of API design.

Reply:
I think, as a general rule, we should not allow the limitations of serde to drive the design, especially for user-visible features.  If from the user perspective the suffix is redundant, I say we drop it. On the backend side, this might be just a matter of having different types for input and output endpoints. @snkas , let's discuss offline.